ui/components/scans/no-permission.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import { InfoIcon } from "lucide-react";
+
+import { CustomButton } from "../ui/custom";
+
+export const NoPermission = () => {
```

Comment:
I suggest to abstract this component in case we need this banner in any other place.
Pass the props:

```suggestion
export const NoPermission = ({title, message}) => {
```

Then:

```tsx
		<div className="flex flex-col space-y-3">
          <div className="flex items-center justify-start gap-3">
            <InfoIcon className="h-6 w-6 text-gray-800 dark:text-white" />
            <h2 className="text-lg font-bold text-gray-800 dark:text-white">
              {{title}}
            </h2>
          </div>
          <p className="text-sm text-gray-600 dark:text-gray-300">
            {{message}}
          </p>
        </div>
```

The add it to the correcto folder in the app

Reply:
Updated